tests/test_planetary_test_data.py

```diff
@@ -7,9 +7,123 @@
 
 Tests for `planetary_test_data` module.
 """
-import pytest
-import planetary_test_data
+import planetary_test_data as planet
+from planetary_test_data import planetary_test_data
+import os
+import re
 
+data_path_1 = os.path.join('tests', 'mission_data', 'data.json')
+data_path_2 = os.path.join('mission_data', 'data.json')
 
-def test_something:
-    assert True == False, 'Write some tests....'
+if os.path.basename(os.getcwd()) == 'tests':
+    pass
+else:
+    os.chdir('tests')
+
+
+def test_default_path():
+    default_path = os.path.abspath(planet.__file__)
+    test_path = os.path.join('planetary_test_data', 'planetary_test_data',
+                             '__init__.py')
+    assert re.search(test_path, default_path)
+
+
+def test_default_json():
+    default_path = os.path.abspath(planetary_test_data.__file__)
+    default_json = os.path.join(os.path.dirname(default_path), 'data.json')
+    test_path = os.path.join('planetary_test_data', 'planetary_test_data',
+                             'data.json')
+    assert re.search(test_path, default_json)
+
+
+def test_final_path_1():
+    # Test when tests/mission_data/data.json exists
+    os.chdir('test_1')
+    assert os.path.exists(data_path_1)
+    assert planetary_test_data.setup_json_file() == data_path_1
+    os.chdir('..')
+    assert os.path.basename(os.getcwd()) == 'tests'
+
+
+def test_final_path_2():
+    # Test when tests/mission_data/ exists but data.json Does Not Exist (DNE)
+    os.chdir('test_2')
+    assert os.path.exists(os.path.join('tests', 'mission_data'))
+    assert planetary_test_data.setup_json_file() == data_path_1
+    os.remove(data_path_1)
+    assert not(os.path.exists(data_path_1))
+    os.chdir('..')
+    assert os.path.basename(os.getcwd()) == 'tests'
```

Comment:
The seemingly excessive asserts is my attempt to test any additional code I needed to run the test